cypress-E2E-tests/cypress/integration/asset_properties_widget.cy.ts

```diff
@@ -334,11 +333,11 @@ describe('Asset Properties Widget: Configuration/View screen tests', function ()
     const assetName = 'check 1';
     cy.get(searchElement).type(assetName);
     cy.get(searchResultsElement).should('have.text', 'Search results');
-    cy.get('c8y-li > div').eq(1).should('contains.text', assetName).click();
+    cy.get('[title="check 1"] > :nth-child(1) > .c8y-list__item__block > .c8y-list__item__body').should('contains.text', assetName).click();
```

Comment:
Please use valid html element instead of css selector

Reply:
added html selector